examples/adminapi.py

```diff
@@ -152,7 +152,16 @@ def example_create_acls(a, args):
         )
     ]
 
-    fs = a.create_acls(acl_bindings, request_timeout=10)
+    fs = None
+    try:
+        fs = a.create_acls(acl_bindings, request_timeout=10)
+    except ValueError as e:
+        print(f"Invalid input: {e}")
+        return
+
+    if fs is None:
```

Comment:
is this possible? I don't reckon, no need to check.

Reply:
done